### PR TITLE
Switch to Particular.Packaging and cleanup

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

**Comment by danielmarbach** [comment]
@Particular/azure-maintainers please review

**Comment by danielmarbach** [comment]
Do we need an upgrade guide for the deprecated AzureConfigurationSource? If yes could someone help me out because I don't really understand what it achieved

**Comment by SeanFeldman** [comment]
> Do we need an upgrade guide for the deprecated AzureConfigurationSource? If yes could someone help me out because I don't really understand what it achieved

Discussed during maintainers sync. Issue raised and moved into `ready` state to be addressed as a part of the bigger doco overhaul.

File: src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj [hunk 1]
```diff
@@ -17,12 +16,15 @@
...
+  </PropertyGroup>
+
   <ItemGroup>
-	<None Include="..\..\packaging\nuget\nservicebus.hosting.azure.hostprocess.nuspec" Link="nservicebus.hosting.azure.hostprocess" />
-    <PackageReference Include="NServiceBus" Version="6.0.0" />
-    <PackageReference Include="Particular.CodeRules" Version="0.2.0" />
-    <PackageReference Include="NuGetPackager" Version="0.6.3" />
-    <PackageReference Include="GitVersionTask" Version="4.0.0-beta0012" />
+    <PackageReference Include="NServiceBus" Version="7.0.0-*" />
+    <PackageReference Include="Particular.CodeRules" Version="*" PrivateAssets="All"/>
```
Comment (SeanFeldman) [inline src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj]: nice catch

**Pull request merged**